.eslintrc

```diff
@@ -22,32 +22,15 @@
   ],
   "root": true,
   "rules": {
-    "no-trailing-spaces": [
-      "error",
-      { "ignoreComments": true }
-    ],
+    "no-trailing-spaces": [ "error", { "ignoreComments": true } ],
     "no-console": "off",
-    "max-len": [
-      1,
-      {
-        "code": 130
-      }
-    ],
-    "semi": [
-      "error",
-      "always"
-    ],
-    "comma-dangle": "off",
+    "max-len": [ 1, { "code": 130 } ],
+    "semi": [ "error", "always" ],
+    // "comma-dangle": ["error", "never"],
+    // "quotes": ["error", "double"],
```

Comment:
Wanted to touch base on this one, I think these two rules make sense, if not I can remove them.

The rest of the changes here are just whitespace